backend/CircleForms/Contracts/V1/ContractModels/Request/PostUpdateRequestContract.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CircleForms.Models;
+using Newtonsoft.Json;
+
+namespace CircleForms.Contracts.V1.ContractModels.Request;
+
+public class PostUpdateRequestContract
+{
+    [JsonProperty("title")]
+    public string Title { get; set; }
+
+    [JsonProperty("description")]
+    public string Description { get; set; }
+
+    [JsonProperty("limitations")]
+    public Limitations Limitations { get; set; }
+}
```

Comment:
Questions should be updatable as well, when the post is not public yet

Reply:
Actually, all of the data should only be updatable when post is not published, unless user is a moderator/admin

Reply:
>Actually, all of the data should only be updatable when post is not published, unless user is a moderator/admin
Right now it's a low-prio feature, because we have no publish stage what so ever

Reply:
Publishing isn't low-prio since editing live posts shouldn't be a thing to begin with. Just keep it in mind down the line, it should definitely come before the initial release.